composer.json

```diff
@@ -215,7 +215,7 @@
       },
       "monaco-editor": {
         "url": "https://registry.npmjs.org/monaco-editor/-/monaco-editor-0.16.2.tgz",
-        "path": "ext/afform/html/bower_components/monaco-editor"
+        "path": "bower_components/monaco-editor"
```

Comment:
Does composer.lock need updating? I'm getting a warning about it being out of date and I'm guessing it's this.

Reply:
@totten ?

Reply:
Yes and no. Changing the `extra.download` section meant that the digest in `composer.lock` became stale, so you get the warning. However, functionally, the lock file isn't relevant to `extra.download`, so you get the expected behavior.

The easy way to cleanup the warning is `composer update --lock`. (I didn't do it at the time because I wasn't sure how long it'd take to merge -- and doing it increases the odds of a pointless merge-conflict.)

Semantically, the `extra.downloads` provides the same guarantees as `composer.lock` (ie locking to a specific download URL), so this is sorta superfluous. For a more permanent solution... the easiest thing might be a patch for `composer-downloads-plugin` to optionally read from an adjacent file (`civicrm-core:composer.downloads.json`). That'd take it out of the scope of `composer.lock` conflicts.

Reply:
Ok thanks that explains it. 🔒